oasst-data/oasst_data/schemas.py

```diff
@@ -11,7 +11,7 @@ class LabelAvgValue(BaseModel):
     count: int
 
 
-LabelValues = dict[str, LabelAvgValue]
```

Comment:
What is the reason for this change (and the similar one for `emojis` below)?

Reply:
`emojis` and `labels` fields has a slightly different structure in loaded HF dataset, for example

original jsonl dataset file
``` json
        "emojis": {
            "+1": 10,
            "_skip_reply": 1,
            "_skip_ranking": 4
        },
```

HF loaded dataset
```json
        "emojis": {
            "name": [
                "+1",
                "_skip_reply",
                "_skip_ranking"
            ],
            "count": [
                10,
                1,
                4
            ]
        },

```

So I did these changes to be possible to parse with pydantic in both scenarios. I though about doing some transformations on HF structure but it would make the loading process slower.